### ISSUE TITLE
Code is not consistent with the error message

### ISSUE THREAD

**Issue by fndjjx** [automl/auto-sklearn#319]
Hi
I found the code is not consistent with the error message, I think the exclude_preprocessors in if condition should be exclude_estimators.

auto-sklearn/autosklearn/util/pipeline.py
line: 28
```
    if include_estimators is not None and \
            exclude_preprocessors is not None:
        raise ValueError('Cannot specify include_estimators and '
                         'exclude_estimators.')
```

auto-sklearn/autosklearn/smbo.py
line: 548
```
        if self.include_estimators is not None and \
                self.exclude_preprocessors is not None:
            raise ValueError('Cannot specify include_estimators and '
                             'exclude_estimators.')
```

**Comment by mfeurer** [comment]
Thanks for reporting this @fndjjx. @KEggensperger did you fix this in #302?

**Comment by mfeurer** [comment]
Was fixed in #302.
